docs/Using-Fleet/FAQ.md

```diff
@@ -69,7 +68,7 @@ It’s standard deployment practice to have multiple Fleet servers behind a load
 
 ## Can I target my hosts using their enroll secrets?
 
-No, currently, there’s no way to retrieve the name of the enroll secret with a query. This means that there's no way to create a label using your hosts' enroll secrets and then use this label as atarget for live queries or scheduled queries.
```

Comment:
Was on the fence about moving this into the enroll secret section. Left it in the FAQ because it is a question about querying as well.